.github/workflows/policy.yml

```diff
@@ -0,0 +1,34 @@
+name: Default Policy
```

Comment:
This is a policy as code file. Not a GitHub Actions workflow file. You might need to place this in a different dir or use the policy in your `10up/.github` repo

Reply:
Moved up a directory level in 6cc130a